### ISSUE TITLE
Handbrake Couldn't find a valid track

### ISSUE THREAD

**Issue by t3dc** [automatic-ripping-machine/automatic-ripping-machine#480]
### Description
I'd just finished successfully ripping a TV series from DVDs. I switched to a new show and began getting the below "Couldn't find a valid track" error.

At first, I thought this was a bad disk. So I went on to the next disk in the series. Same error. I thought perhaps it was something odd with the way that series was mastered and moved onto a completely different tv show. Same issue. When I got the error on yet a third series I decided something must have gone pear shaped with my ARM or my drive.

I'm not sure where to dig, though. Handbrake recommends running the command manually to troubleshoot. How can I disable arms auto processing to try that? Has anyone else seen this?

Thanks!

### Environment
Ubuntu 20.04.4 LTS (running under VirtualBox 5.2.22 on Nethserver 7.9.2009)
ARM v2_devel 1337-server fork

### Log file
```
[01-03-2022 20:05:07] INFO ARM: handbrake.get_track_info Using HandBrake to get information on all the tracks on the disc.  This will take a few minutes...
[01-03-2022 20:05:07] DEBUG ARM: handbrake.get_track_info Sending command: HandBrakeCLI -i /dev/sr0 -t 0 --scan
[01-03-2022 20:05:18] ERROR ARM: handbrake.get_track_info Couldn't find a valid track. Try running the command manually to see more specific errors.
[01-03-2022 20:05:18] ERROR ARM: handbrake.get_track_info Specific error is: Command 'HandBrakeCLI -i /dev/sr0 -t 0 --scan' returned non-zero exit status 2.
[01-03-2022 20:05:28] ERROR ARM: handbrake.get_track_info Couldn't find a valid track. Try running the command manually to see more specific errors.
[01-03-2022 20:05:28] ERROR ARM: handbrake.get_track_info Specific error is: Command 'HandBrakeCLI -i /dev/sr0 -t 0 --scan' returned non-zero exit status 2.
[01-03-2022 20:05:28] DEBUG ARM: handbrake.handbrake_all Total number of tracks is None
```

**Comment by 1337-server** [comment]
Have you tried running `HandBrakeCLI -i /dev/sr0 -t 0 --scan` with a disc in ? Sometimes HandBrake will find them, but ARM will fail to handle them correctly.
The above command should help us narrow the problem down.

**Comment by t3dc** [comment]
Hi 1337-server,

Thanks for the reply. I was trying to figure out how to do that without ARM grabbing the disk. However, in the interim, I tried switching from the backup to the mkv setting to bypass Handbrake entirely. Using MakeMKV, I get the dreaded "READ ERROR" (http://www.makemkv.com/errors/read/) -- which they indicate is a failed drive (since I've tried with multiple discs). If you've experience to the contrary, I'd love to hear it, but sounds like I've lost another reader. Anyone have a good source for buying drives in a post-optical world?
